src/main/java/seedu/club/logic/commands/DeleteCommand.java

```diff
@@ -47,12 +49,12 @@ public CommandResult executeUndoableCommand() throws CommandException {
         requireToLogIn();
         requireExcoLogIn();
         try {
-            model.deleteMember(memberToDelete);
+            numberOfTasksDeleted = model.deleteMember(memberToDelete);
```

Comment:
I see what you're trying to do here.
But, should `deleteMember` return `numOfTaksDeleted`?

Reply:
strictly speaking, it shouldn't. however, there are a lot of implementation overheads especially in terms of testing if i were to use another method.